pint/solar_system_ephemerides.py

```diff
@@ -163,3 +163,46 @@ def objPosVel(obj1, obj2, t, ephem):
         return obj2pv - obj1pv
     else:
         return PosVel(np.zeros((3,len(t)))*u.km, np.zeros((3,len(t)))*u.km/u.second)
+
+def get_tdb_tt_ephem(t, ephem, path=None, link=None):
+    """The is a function to read the TDB_TT correction from the JPL DExxxt.bsp
+       ephemeris file.
```

Comment:
There seems no link for the tt-tdb ephemeris definition. There may be some pdf or txt documents. But the comments in the ephemeris have the definition of segments, which has part of TDB-TT.
If you do:
```
>>> from jplephem.spk import SPK
>>> kernel = SPK.open('de436t.bsp')
>>> print(kernel)
```
It should have the information you need.